module/VuFind/src/VuFind/ILS/Driver/Alma.php

```diff
@@ -1878,21 +1924,87 @@ protected function getTranslatableStatusString($element)
         }
         $value = 'status_' . strtolower((string)$element);
         $desc = (string)($element->attributes()->desc ?? $element);
-        return new \VuFind\I18n\TranslatableString($value, $desc);
+        return new TranslatableString($value, $desc);
     }
 
     /**
      * Get location for an item
      *
      * @param SimpleXMLElement $item Item
      *
-     * @return \VuFind\I18n\TranslatableString|string
+     * @return TranslatableString|string
      */
     protected function getItemLocation($item)
     {
         return $this->getTranslatableString($item->item_data->location);
     }
 
+    /**
+     * Get location type for an item
+     *
+     * @param SimpleXMLElement $item Item
+     *
+     * @return string
+     */
+    protected function getItemLocationType($item)
```

Comment:
There were existing unit tests only for Alma resolver driver, but since adding new tests made sense I went through the effort here. Now all of getHolding and the new methods should be covered. While working on it I noticed that the resulting URLs were a bit funny when paging wasn't enabled, so that's fixed as well. And there was a useless constructor param (config loader), which was removed.

Reply:
Thanks for taking the time!